### PR TITLE
[ISSUE-#3855] Provided a feature to view version comparisons in the configDetails page

### PR DESCRIPTION
**Pull request by jzhishu**
## What is the purpose of the change

#3855

## Brief changelog

- In the configDetails page, you can click the "Version Comparison" button to open the configuration content version comparison

### CODE REVIEW

**Review by KomachiSion — APPROVED** [review]

**Comment by KomachiSion** [comment]
@jzhishu Can you add some unit test or integration test for your changes?

**Comment by jzhishu** [comment]
> @jzhishu Can you add some unit test or integration test for your changes?

Ok, I'll add the test case later

File: config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java [hunk 1]
```diff
@@ -72,12 +72,20 @@
...
+     * Query the detailed configuration history information.
      */
     @GetMapping
     public ConfigHistoryInfo getConfigHistoryInfo(HttpServletRequest request, HttpServletResponse response,
             @RequestParam("nid") Long nid, ModelMap modelMap) {
         return persistService.detailConfigHistory(nid);
     }
     
+    /**
+     * Query previous config history information.
```
Comment (KomachiSion) [inline config/src/main/java/com/alibaba/nacos/config/server/controller/HistoryController.java:84]: Please add `@since 1.4.0`

**Review by KomachiSion — CHANGES_REQUESTED** [review]

**Review by KomachiSion — DISMISSED** [review]

**Review by loadchange — APPROVED** [review]

**Comment by chuntaojun** [comment]
Can you add some pictures to show the final form

**Comment by jzhishu** [comment]
image
image

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**